tidb-lightning: clarify an ambiguous description for `sorted-kv-dir` (#7962)
pingcap/docs: migrate-large-mysql-shards-to-tidb.md

    # For more information, see [TiDB Lightning Backends](https://docs.pingcap.com/tidb/stable/tidb-lightning-backends)
    backend = "local"
    # Set the temporary directory for the sorted key value pairs. It must be empty.

>     # The free space must be large enough to hold the dataset to be imported.

Author: ```suggestion
    # The free space must be greater than the size of the dataset to be imported.
```
